.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+<!--- Add the issue this PR addresses -->
+Fixes https://github.com/gradle/gradle/issues/
```

Comment:
See: https://help.github.com/en/articles/about-issue-and-pull-request-templates#pull-request-templates

Reply:
I think if we want this it should say `Fixes #`, because that way you get the nice popup with the issues as you start typing.

![image](https://user-images.githubusercontent.com/495366/60330569-0c78db00-9993-11e9-8eb4-6e30db391fa2.png)

Reply:
```suggestion
Fixes #
```